.env.development

```diff
@@ -7,7 +7,7 @@
 # feature flags
 VITE_FEATURE_THORCHAIN_TCY_ACTIVITY=true
 VITE_FEATURE_QUICK_BUY=true
-VITE_FEATURE_NEW_WALLET_MANAGER=true
+VITE_FEATURE_NEW_WALLET_MANAGER=false
```

Comment:
Make sure to revert me before merging. This is disabled in code here and in code, since new wallet manager still has a few shenanigans.

Reply:
Thanks to my fresh self for that comment or I'd totally merge that and rug you on that feature a second time around @premiumjibles